app/views/course/forum/posts/_post.html.slim

```diff
@@ -0,0 +1,27 @@
+a.post-anchor name="post-#{post.id}"
+div.post id="post-#{post.id}"
+  div.user
+    h5 = link_to_user(post.creator)
+
+  div.contents
+    div.pull-right
+      .timestamp
+        => format_datetime(post.created_at)
+
+    h4.title
+      a href="#post-#{post.id}"
+        = post.title
+
+    = post.text
+
+    - unless defined?(hide_button)
```

Comment:
Because this partial can be used to show the posts and reply to a specific post. When you show the posts, it should provide buttons like 'reply, delete' and so on. But in reply page, this partial is used to show a specific post without any buttons.